include/fastdds/rtps/attributes/HistoryAttributes.h

```diff
@@ -0,0 +1,90 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file HistoryAttributes.h
```

Comment:
Added DOXYGEN_SHOULD_SKIP_THIS_PUBLIC

Reply:
That solution doesn't work. I've researched a bit on the internet and I think the only way to solve this is to change the @file directives with more complete paths. In this file it would be:
`@file include/fastdds/rtps/attributes/HistoryAttributes.h`

However, as you commented to me, fastrtps files will become obsolete eventually. We will review the documentation then.